lib/api/core/models/repositories/tokenRepository.js

```diff
@@ -221,6 +221,7 @@ class TokenRepository extends Repository {
       emptyKeyLength = super.getCacheKey('').length,
       userKey = super.getCacheKey(userId + '#*');
 
+    // @todo ask @scottinet or @bvidis for this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)